cms/static/js/features/import/factories/import.js

```diff
@@ -44,7 +44,7 @@ define([
                         $('.progress').show();
                     } else {
                         msg = gettext('File format not supported. Please upload a file with a {ext} extension.')
-                            .replace('{ext}', '<code>tar.gz</code>');
+                            .replace('{ext}', 'tar.gz');
 
                         $('.error-block').text(msg).show();
```

Comment:
instead of removing the tag use `.html()` instead of `.text()`

Reply:
Done